### PR TITLE
fix: Type hinting for the pad_batch mode argument

### PR DESCRIPTION
**Pull request by Apalian**
Correction of the `mode` argument of the `mfai.pytorch.padding.pad_batch()` function so that it is of type `Literal` with the value of the `mode` argument of `torch.nn.functional.Pad`.

### CODE REVIEW

**Comment by LBerth** [comment]
Hi ! Could you provide a more precise title for the PR, and link the related issue ? :)

**Review by kalharko — APPROVED** [review]

**Review by LBerth — APPROVED** [review]
